WordPressVIPMinimum/Sniffs/Compatibility/RestrictedCacheGroupSniff.php

```diff
@@ -0,0 +1,154 @@
+<?php
+/**
+ * WordPressVIPMinimum Coding Standard.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ */
+
+namespace WordPressVIPMinimum\Sniffs\Compatibility;
+
+use WordPressCS\WordPress\AbstractFunctionParameterSniff;
+use PHP_CodeSniffer\Util\Tokens;
+
+/**
+ * Check if cache group name is already in use by wp-memcached plugin to avoid naming collision.
+ *
+ * Note: wp-memcached is automatically enabled on WordPress VIP.
+ *
+ * @package VIPCS\WordPressVIPMinimum
+ *
+ * @since 2.4.0
+ */
+class RestrictedCacheGroupSniff extends AbstractFunctionParameterSniff {
+
+	/**
+	 * The group name for this group of functions.
+	 *
+	 * @var string
+	 */
+	protected $group_name = 'wp_cache_functions';
+
+	/**
+	 * Functions that can assign or modify cache key.
+	 *
+	 * @var array <string function name> => <bool (true)>
+	 */
+	protected $target_functions = [
```

Comment:
@jrfnl It should include it since we don't want users touching that cache group at all. However, since `wp_cache_delete()` uses a different parameter to check for, should I modify the logic to account for that if I add that function to the list?

Reply:
You could change the value in the array to contain the parameter position instead of `true` and yes, then modify the sniff to use the position number to find the right parameter to examine.

I think there should be some sniffs around (either in VIPCS or in WPCS) which already do so if you'd like to look at a code pattern for this.